lib/api/core.js

```diff
@@ -0,0 +1,55 @@
+import { Meteor } from "meteor/meteor";
+import { Shops } from "/lib/collections";
+import { SellerShops } from "/imports/plugins/included/marketplace/lib/collections";
```

Comment:
Yes. I think we'll add that check function in here directly and remove these imports